prow/cluster/hook_deployment.yaml

```diff
@@ -30,10 +30,10 @@ spec:
       labels:
         app: hook
     spec:
-      terminationGracePeriodSeconds: 30
+      terminationGracePeriodSeconds: 180
```

Comment:
> When kubelet sends SIGTERM to a pod, the pod is marked with a deletionTimestamp 

Not 100% sure but the series of events should be:
* api server marks pod with deletion timestamp
* kubelet notices deletion timestamp is set and sends SIGTERM to pod
* after grace period passes, kubelet sends SIGKILL. This is fuzzy, I think the pod may be removed prior or even after the grace period passes.